core/shared/src/main/scala/result.scala

```diff
@@ -36,7 +41,7 @@ class Catching[E <: Exception]() {
   }
 }
 
-sealed class Result[+T, E <: Exception](val answer: T, val errors: Seq[(ClassTag[_], (String, Exception))],
+sealed abstract class Result[+T, +E <: Exception](val answer: T, val errors: Seq[(ClassTag[_], (String, Exception))],
```

Comment:
Oh, just noticed you've changed the variance on `E`. I'm not sure this will work...